gas-station/typescript/src/sponsored_examples.ts

```diff
@@ -1,163 +1,240 @@
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     const [coin1, coin2] = txb.splitCoins(txb.object(COIN_TO_SPLIT), [
+//       txb.pure(10000),
+//       txb.pure(20000),
+//     ]);
+//     // each new object created in a transaction must be sent to an owner
+//     txb.transferObjects([coin1, coin2], txb.pure(SENDER_ADDRESS));
+//   }
+// });
+
+
+
+//  Transfer one or more object(s) owned by the sender address to the recipient
+//    An easy example is a small coin you created with the above transaction.
+//
+//
+// const RECIPIENT_ADDRESS = "{{recipientSuiAddress}}";
+// const OBJECT_TO_SEND = "{{objectId}}";
+// const gaslessPayloadBase64 = await buildGaslessTransactionBytes({
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.transferObjects(
+//       [txb.object(OBJECT_TO_SEND)],
+//       txb.pure(RECIPIENT_ADDRESS)
+//     );
+//   }
+// });
+
+
+
+//  Merge one or more smaller coins into another, destroying the small coin(s)
+//    and increasing the value of the large one.
+//
+//
+// const TARGET_COIN = "{{targetCoin}}";
+// const COIN_SOURCE1 = "{{coinSource1}}";
+// const COIN_SOURCE2 = "{{coinSource2}}";
+// const gaslessPayloadBase64 = await buildGaslessTransactionBytes({
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.mergeCoins(txb.object(TARGET_COIN), [
+//       txb.object(COIN_SOURCE1),
+//       txb.object(COIN_SOURCE2),
+//     ]);
+//   }
+// });
+
+
+// -- Invisible Wallet: sign, sponsor, and execute as separate steps -- //
+
+// 3. Use a pre-existing Shinami Invisible Wallet as the sender
+// import { KeyClient, WalletClient } from "@shinami/clients";
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// const walletClient = new WalletClient(WALLET_SERVICES_KEY);
+// const SENDER_ADDRESS = await walletClient.getWallet(INVISIBLE_WAL_ID);
+
+// 8. Sign the full transaction payload with the sender's key.
+// const keyClient = new KeyClient(WALLET_SERVICES_KEY);
+
+// const sessionToken = await keyClient.createSession(INVISIBLE_WAL_SECRET);
+// const senderSig = await walletClient.signTransactionBlock(
+//   INVISIBLE_WAL_ID,
+//   sessionToken,
+//   sponsoredResponse.txBytes
+// );
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// -- Invisible Wallet: sponsor, sign, and execute in one method call -- //
+// import { 
+//   KeyClient, 
+//   WalletClient, 
+//   createSuiClient, 
+//   buildGaslessTransactionBytes 
+// } from "@shinami/clients";
+
+// // 1. Copy your Node and Wallet Services Testnet Api Key Values
+// const NODE_TESTNET_ACCESS_KEY = "sui_testnet_bb0ae4d25b7e51a98c8fcb267babc1b6";
+// const WALLET_SERVICES_ONLY_KEY = "3ccf23b29a99cb4d4c2974e590575f23";
+// const ALL_SERVICES_TESTNET_KEY = "sui_testnet_a683fdb22331e5a9ef146b4dc2f5cf67";
```

Comment:
Remove these and destroy the keys as well

Reply:
Done.